CHANGELOG.md

```diff
@@ -0,0 +1,76 @@
+## Unreleased
+
+No changes yet :)
```

Comment:
What this line stands for?

Reply:
Eventually this section will have the changes which are in develop but not part of any release. For now it's empty, this is just a placeholder so the empty section doesn't look weird, it will be removed once a PR is merged.